src/tests/ftest/checksum/basic_checksum.py

```diff
@@ -2,24 +2,7 @@
 """
   (C) Copyright 2019 Intel Corporation.
```

Comment:
You should install the githook and refresh this...it will update they copyrights

Reply:
Do we have githook scripts or package to do the Copyright year update? or I need to create something like pre-coomit in daos/.git/hooks/

Reply:
linked to utils/githooks/pre-commit, but .py files failed to get the update. Plan to leave .py files as they are now and expect get updated when utils/githooks/pre-commit is updated.